composer.json

```diff
@@ -24,7 +29,10 @@
     "dpc-sdp/tide_core": "^3.0.0",
     "dpc-sdp/tide_landing_page": "^3.0.0",
     "drupal/elasticsearch_connector": "^7.0",
-    "drupal/search_api": "^1.11"
+    "drupal/search_api": "^1.11",
+    "drupal/data_pipelines": "^1.0@alpha",
+    "drupal/data_pipelines_elasticsearch": "^1.0@alpha",
```

Comment:
Note there may be an updated version coming from my fix for D10 bulk endpoint, or we may have to add another patch if it's not released in time.

Reply:
Cool, I will keep an eye while updating to drupal 10, thanks for the heads up